packages/terra-application-header-layout/tests/jest/ApplicationHeader.test.jsx

```diff
@@ -1,121 +1,175 @@
 import React from 'react';
+import { IntlProvider } from 'react-intl';
```

Comment:
You can import this from terra-base, and then you can remove the devDependency on react-intl.